server/views/pages/reviewPlan/checkYourAnswers.njk

```diff
@@ -12,7 +12,75 @@
 
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-        <h1 class="govuk-heading-l">Check your answers</h1>
+      <h1 class="govuk-heading-l" data-qa="page-heading">Check
+        {{prisonerSummary.firstName + " " + prisonerSummary.lastName}}'s review information</h1>
+
+      <div class="app-u-heading-with-change-link">
+        <h2 class="govuk-heading-m">
+          {% if reviewPlanDto.completedBy === 'MYSELF' %}
+            Who completed the review?
+          {% else %}
+            Who completed
+            {{ prisonerSummary.firstName + " " + prisonerSummary.lastName }}'s review?
+          {% endif %}
+        </h2>
+        <span class="govuk-body-m">
+          <a class="govuk-link" href="/plan/{{ prisonerSummary.prisonNumber }}/review" data-qa="review-completed-by-change-link" style="white-space: nowrap;">
+            Change<span class="govuk-visually-hidden">
+              who completed the review</span>
+          </a>
+        </span>
+      </div>
+
+      <dl class="govuk-summary-list govuk-!-margin-bottom-8">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key govuk-!-width-one-quarter">Review completed by</dt>
+          <dd class="govuk-summary-list__value" data-qa="review-completed-by">
+            {% if reviewPlanDto.completedBy === 'MYSELF' %}
+              {{ userName }}
+            {% else %}
+              {{ reviewPlanDto.completedByOtherFullName }}
+            {% endif %}
+          </dd>
+        </div>
+        {% if reviewPlanDto.completedBy === 'SOMEBODY_ELSE' %}
+          <div class="govuk-summary-list__row">
+            <dt class="govuk-summary-list__key govuk-!-width-one-quarter">Job role</dt>
+            <dd class="govuk-summary-list__value" data-qa="job-role">
+              {{ reviewPlanDto.completedByOtherJobRole }}
+            </dd>
+          </div>
+        {% endif %}
+      </dl>
+      <div class="app-u-heading-with-change-link">
+        <h2 class="govuk-heading-m">Review note</h2>
+        <span class="govuk-body-m">
+          <a class="govuk-link" href="/plan/{{ prisonerSummary.prisonNumber }}/review/notes" data-qa="review-note-change-link" style="white-space: nowrap;">
+            Change<span class="govuk-visually-hidden">
+              who completed the review</span>
+          </a>
+        </span>
+      </div>
+      <dl class="govuk-summary-list govuk-!-margin-bottom-8">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key govuk-!-width-one-quarter">
+            {% if reviewPlanDto.completedBy === 'MYSELF' %}
+              Note added
```

Comment:
As above; do we really have different heading/label text for the note, depending on who added it?
Lets check this with the designers

Reply:
Confirmed correct with Karen.